src/proto/thread_telemetry.proto

```diff
@@ -339,6 +339,34 @@ message TelemetryData {
     optional SrpServerResponseCounters response_counters = 6;
   }
 
+  message TrelPacketCounters {
+    // The number of packets successfully transmitted through TREL
+    optional uint64 trel_packets_tx = 1;
+
+    // The number of bytes successfully transmitted through TREL
+    optional uint64 trel_bytes_tx = 2;
+
+    // The number of packets transmission failures through TREL
+    optional uint64 trel_packets_tx_failed = 3;
+
+    // The number of packets successfully received through TREL
+    optional uint64 trel_packets_rx = 4;
+
+    // The number of bytes successfully received through TREL
+    optional uint64 trel_bytes_rx = 5;
```

Comment:
The field names are different from those in `types.hpp`, i.e. 'tx' and 'rx' are put to the end of the names. What's the idea behind this?

Reply:
Renamed names in proto to put tx in the middle.